tcmu-runner.spec

```diff
@@ -1,3 +1,26 @@
+%global _hardened_build 1
+
+# without rbd dependency
+# if you wish to exclude rbd handlers in RPM, use below command
+# rpmbuild -ta @PACKAGE_NAME@-@PACKAGE_VERSION@.tar.gz --without rbd
+%{?_without_rbd:%global _without_rbd -Dwith-rbd=false}
+
+# without glusterfs dependency
+# if you wish to exclude glfs handlers in RPM, use below command
+# rpmbuild -ta @PACKAGE_NAME@-@PACKAGE_VERSION@.tar.gz --without glfs
+%{?_without_glfs:%global _without_glfs -Dwith-glfs=false}
+
+# without qcow dependency
+# if you wish to exclude qcow handlers in RPM, use below command
+# rpmbuild -ta @PACKAGE_NAME@-@PACKAGE_VERSION@.tar.gz --without qcow
+%{?_without_qcow:%global _without_qcow -Dwith-qcow=false}
+
+# without zbc dependency
+# if you wish to exclude zbc handlers in RPM, use below command
+# rpmbuild -ta @PACKAGE_NAME@-@PACKAGE_VERSION@.tar.gz --without zbc
+%{?_without_zbc:%global _without_zbc -Dwith-zbc=false}
+
```

Comment:
Another option was added the other day for file_optical.

Reply:
Thanks for pointing this out. I checked it out, and currently there is no option to provide a cmake option for that through `CMakeLists.txt` file. Hence I am thinking I should do it as a separate PR to provide option in cmake, and then option in rpm and make_runnerrpms.sh files.

If that if fine, I will create an issue, and work on that separately. So, this patch can be merged as is.

Reply:
We really just added it the other day, so maybe you need to do a repull/clone. See this commit:

https://github.com/open-iscsi/tcmu-runner/commit/daa00063045fafdfc25a13a3d94e2ea77c8a1c3d

Reply:
Ah you know what, it is called fbo (file backed optical) in the cmakelist file which is the abbreviation used for it in the file_optical.c file and what is used for its identifier, so that is probably the confusion.

Reply:
Oops, sorry about not being on tip of master. Source of all confusion. Added the option, and pushed again.